lib/releaselib.sh

```diff
@@ -909,7 +909,7 @@ release::docker::release () {
   else
     # TODO: remove this when all kubernetes releases produce a release-images
     # directory
-    [[ "$registry" =~ gcr.io/ ]] && docker_push_cmd=("$GCLOUD" "docker" "--")
```

Comment:
Looking at this for the first time in a while, it looks like docker_push_cmd can always be "$GCLOUD" "docker" "--", so better to just remove this and set it to that value in the declaration above.

Reply:
ACK